src/pm-conversations/PmConversationList.js

```diff
@@ -29,8 +29,8 @@ export default class PmConversationList extends PureComponent<Props> {
     this.props.dispatch(doNarrow(privateNarrow(email)));
   };
 
-  handleGroupNarrow = (email: string) => {
-    this.props.dispatch(doNarrow(groupNarrow(email.split(','))));
+  handleGroupNarrow = (users: UserOrBot[]) => {
+    this.props.dispatch(doNarrow(groupNarrow(users.map(s => s.email))));
```

Comment:
The order here matters, because we use it for the key to look up the narrow in our data structures. So it's important to preserve the order used by the existing code, or else to carefully untangle to make it stop mattering.

Reply:
Also, doesn't this now see the full set of users? Where previously it saw the set excluding self.

Reply:
That's odd – I specifically remember seeing that issue and fixing it. (The second one, not the first; the first is a carryover of my mistake in describing `recipients`.) Will fix again.